tests/deal_ticket/test_trading_deal_ticket_submit_account.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from playwright.sync_api import Page, expect
+from vega_sim.service import VegaService
+from actions.utils import wait_for_toast_confirmation
+
+
+
+order_size = "order-size"
+order_price = "order-price"
+place_order = "place-order"
+deal_ticket_warning_margin = "deal-ticket-warning-margin"
+deal_ticket_deposit_dialog_button = "deal-ticket-deposit-dialog-button"
+
+@pytest.mark.usefixtures("page", "auth", "risk_accepted")
+def test_should_display_info_and_button_for_deposit(continuous_market, vega: VegaService, page: Page):
+    page.goto(f"/#/markets/{continuous_market}")
+    page.get_by_test_id(order_size).type("200000")
```

Comment:
i think .fill() is safer:
![image](https://github.com/vegaprotocol/console-test/assets/133853298/d57697d3-2f95-48ac-ac52-ea5b26c545c9)

Reply:
done